### PR TITLE
project made it

### PR DESCRIPTION
**Pull request by alieftekhar33**
### **The following requirement added:**

- [ ] Apply best practices in HTML code.
- [ ] Use semantic HTML tags.
- [ ] Use CSS selectors correctly.
- [ ] Use CSS box model.
- [ ] Perform a code review for yourself on a past project.

### CODE REVIEW

**Comment by alieftekhar33** [comment]
**That's nice that changed the readme!**

**Comment by alieftekhar33** [comment]
**your project is licensed by you and makes professional your program.**

**Comment by alieftekhar33** [comment]
**`It's okay and time to merge!`**

**Review by anagudelogu — CHANGES_REQUESTED** [review]
# STATUS:  CHANGES REQUIRED  ♻️ 🚧 

Hi @alieftekhar33, 
I'm Andres.

While you have done a great job working on this milestone 💻 , there are some issues that you still need to work on to go to the next project. ♻️ 

You are almost there, keep up the good work! 🚀 



## Highlights
✅ Descriptive PR title, summary and commit messages. 💬 

## Required Changes ♻️

- [ ] I noticed that you added some comments to the pull request! 💯 However, please read this [lesson](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md) about the GitHub review features and use them! 🤓 

- [ ] I noticed that in the comments you're highlighting things such as the good use of the README file or the LICENSE and it's great! However, the main goal of this milestone is that you learn how to give a code review to another developer. Please consider looking for things to improve in your codebase and ask for changes using the GitHub features! 🤓 🛠️


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: babel/index.babel [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    from red to green.
+  - As you start to build out your 
+    project, when tests are failing, 
+    you should get helpful errors 
+    along the way!
+    ************/
+
+// PLEASE NOTE: Adding global style rules using the * selector, or by adding rules to body {..} or html {..}, or to all elements within body or html, i.e. h1 {..}, has the potential to pollute the test suite's CSS. Try adding: * { color: red }, for a quick example!
+
+// Once you have read the above messages, you can delete all comments.
```
Comment (alieftekhar33) [inline babel/index.babel:27]: ### **Required changes needed**

- [x] - please delete these lines as the instruction

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @alieftekhar33 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

## Highlights

 ✅ Descriptive PR title, summary and commit messages. 💬

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

 - [OPTIONAL] Your comment is polite, however, consider including an explanation, a "why", for each requested change. For example:
> This is a well-explained request: Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible
> 
> This request lacks a "why": Why don't you use an article tag here instead of a div?


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**